src/App.tsx

```diff
@@ -5,7 +5,13 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const handleSetMovies = (movie: Movie) => {
+    if (!movies.find(film => movie.imdbId === film.imdbId)) {
```

Comment:
maybe better some method it will return true if find same movie and false if not